accepted/color-4-new-spaces.md

```diff
@@ -1141,26 +1141,17 @@ The procedure is:
     * Let `split-last` be the result calling `string.split()` with the last
       element of `input` as the string to split, and `/` as the separator.
 
-    * If `split-last` has two items, and one or both items are an unquoted
-      string that's case-insensitively equal to 'none':
+    * If `split-last` doesn't have exactly two items, throw an error.
```

Comment:
Is this meant to deal with the following case?
```css
rgba(1 2 3 calc(var(--x)/var(--x))/var(--x))
```
IIUC by this point `components == [1, 2, 3]` and `split-last == [calc(var(--x), var(--x)), var(--x)]` which would fail, however I think it should technically be permitted 🤔

Reply:
Hmm, that's an interesting point. I think you're right that allowing that is more important than disallowing `rgb(1 2 3 a/b/c/)`.